src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -1654,6 +1654,18 @@ module MutRecBindingChecking =
              mBinds 
         
         defnsEs, envMutRec
+        
+let private CheckStaticClassElements (synMembers: SynMemberDefn list) =
+    for mem in synMembers do
+        match mem with
+        | SynMemberDefn.ImplicitCtor(ctorArgs= SynSimplePats.SimplePats(pats= pats)) when  (not pats.IsEmpty) ->
+            for pat in pats do
+                errorR(Error(FSComp.SR.chkErrorOnStaticClasses "constructor arguments are", pat.Range))
```

Comment:
```suggestion
                errorR(Error(FSComp.SR.chkErrorOnStaticClasses "constructor with arguments", pat.Range))
```

Reply:
I have updated to use 2 different error numbers so this is not longer the case :)